solr/contrib/ltr/src/test/org/apache/solr/ltr/TestRerankBase.java

```diff
@@ -333,7 +333,7 @@ public static LTRScoringModel createModelFromFiles(String modelFileName,
 
     final ManagedFeatureStore fs = getManagedFeatureStore();
     // fs.getFeatureStore(null).clear();
-    fs.doDeleteChild(null, "*"); // is this safe??
+    fs.doDeleteChild(null, (featureStoreName == null ? "_DEFAULT_" : featureStoreName)); // is this safe??
```

Comment:
should use DEFAULT_FEATURE_STORE_NAME from the Managed resource instead of hard coding the string